### PR TITLE
Table names change, TestHistory -> Test and Test -> MotherTest

### PR DESCRIPTION
**Pull request by Jnegrier**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -806,8 +806,8 @@ def get_tests_history_by_test_run(test_run_id):
             test_suites[test_suites_index.get(test_suite_history.id)]["tests"].append(
                 {
                     "test_history_id": test_history.id,
-                    "test_id": test_history.test.id,
-                    "name": test_history.test.name,
+                    "mother_test_id": test_history.mother_test_id,
+                    "name": test_history.mother_test.name,
```
Comment (opishcheiko) [inline app.py:810]: thought about this song:
https://www.youtube.com/watch?v=ZXg9UFUXFXU

File: app.py [hunk 2]
```diff
@@ -806,8 +806,8 @@ def get_tests_history_by_test_run(test_run_id):
             test_suites[test_suites_index.get(test_suite_history.id)]["tests"].append(
                 {
                     "test_history_id": test_history.id,
-                    "test_id": test_history.test.id,
-                    "name": test_history.test.name,
+                    "mother_test_id": test_history.mother_test_id,
```
Comment (opishcheiko) [inline app.py:809]: it's cause you can use it directly from the column instead of mother_test.id, right?

**Comment by opishcheiko** [comment]
thought of this song https://www.youtube.com/watch?v=ZXg9UFUXFXU
👏 , i think our production will blow up

File: app.py (hunk 2 above)
Comment (Jnegrier) [reply app.py:809]: Yep, exactly, we have the value there directly :)

**Comment by Jnegrier** [comment]
> thought of this song https://www.youtube.com/watch?v=ZXg9UFUXFXU
> 👏 , i think our production will blow up, soo cool with new names
> I can do the frontend refactor

I didn't know the song, just added to my playlist 😃 
In theory, it should be okay, I'm doing a lot of tests now, also, I'm already working in the frontend, no worries. I'll need your expert eyes when checking the PR 🦅

**Comment by Jnegrier** [comment]
@opishcheiko Ready for review 👍

**Review by opishcheiko — APPROVED** [review]
🤞

**Comment by opishcheiko** [comment]
should we leave endpoint renaming to the later stage?

**Comment by Jnegrier** [comment]
> should we leave endpoint renaming to the later stage?

Yep, maybe is the best, has that will impact on the clients as well. Also, we could add the cascade relation in that iteration

**Pull request merged**